src/TransactionBuilder.php

```diff
@@ -36,15 +36,16 @@ class TransactionBuilder
      * @param string        $companyCode   The code of the company for this transaction
      * @param DocumentType  $type          The type of transaction to create (See DocumentType::* for a list of allowable values)
      * @param string        $customerCode  The customer code for this transaction
+     * @param string|null   $dateTime      The datetime of the transaction, defaults to current time when null (Format: Y-m-d H:i:s)
      */
-    public function __construct($client, $companyCode, $type, $customerCode)
+    public function __construct($client, $companyCode, $type, $customerCode, $dateTime = null)
     {
         $this->_client = $client;
         $this->_line_number = 1;
         $this->_model = [
             'companyCode' => $companyCode,
             'customerCode' => $customerCode,
-            'date' => date('Y-m-d H:i:s'),
+            'date' => $dateTime !== null ? $dateTime : date('Y-m-d H:i:s'),
```

Comment:
@jondmcelroy 
Thank you for the changes, the date field should be a string containing the date only, i.e 2018-03-01.
https://developer.avalara.com/api-reference/avatax/rest/v2/models/CreateTransactionModel/
Please remove the H:i:s and resubmit the pull request.

Thank you,
Avalara Developer Team